SubEthaEdit-Mac/Source/TCMFoundation/TCMBencodingUtilities.m

```diff
@@ -148,7 +144,7 @@ void TCM_AppendBencodedObjectToData(id inObject, NSMutableData *inData) {
 // returns an autoreleased object
 id TCM_BdecodedObjectWithData(NSData *data) {
     unsigned position=0;
-    return [TCM_CopyBdecodedObject((uint8_t *)[data bytes],&position,[data length]) autorelease];
+    return TCM_CopyBdecodedObject((uint8_t *)[data bytes],&position,[data length]);
 }
 
 
```

Comment:
`TCM_CopyBdecodedObject` needs a CF_RETURNS_RETAINED to signify that it doesn't return a autoreleased object to be compatible in non-arc environments.